src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerWaitContainer.groovy

```diff
@@ -29,20 +29,20 @@ class DockerWaitContainer extends DockerExistingContainer {
      */
     @Input
     @Optional
-    Long timeout
+    Integer timeout
 
     DockerWaitContainer() {
         ext.getExitCode = { exitCode }
     }
 
     @Override
     void runRemoteCommand(dockerClient) {
-        String possibleTimeout = timeout ? " for ${timeout} milliseconds" : ''
+        String possibleTimeout = timeout ? " for ${timeout} seconds" : ''
         logger.quiet "Waiting for container with ID '${getContainerId()}'${possibleTimeout}."
         def containerCommand = dockerClient.waitContainerCmd(getContainerId())
         def callback = threadContextClassLoader.createWaitContainerResultCallback()
         callback = containerCommand.exec(callback)
-        exitCode = timeout ? callback.awaitStatusCode(timeout, TimeUnit.SECONDS) : callback.awaitStatusCode()
+        exitCode = timeout ? callback.awaitStatusCode((long)timeout, TimeUnit.SECONDS) : callback.awaitStatusCode()
```

Comment:
Casting to long should be done automagically. Any particular reason to keep this in?